onedal/datatypes/data_conversion.cpp

```diff
@@ -237,7 +237,7 @@ template <int NpType, typename T = byte_t>
 static PyObject *convert_to_numpy_impl(const dal::array<T> &array,
                                        std::int64_t row_count,
                                        std::int64_t column_count = 0) {
-    const std::int64_t size_dims = column_count == 0 ? 1 : 2;
+    int size_dims = column_count == 0 ? 1 : 2;
```

Comment:
Required to be int type here: https://numpy.org/devdocs/reference/c-api/array.html#c.PyArray_SimpleNewFromData

Reply:
Can it still be `const` ?